packages/config/__tests__/ProfileInfo.TeamConfig.test.ts

```diff
@@ -1096,19 +889,21 @@ describe("ProfileInfo tests", () => {
             const mergedArgs = profInfo.mergeArgsForProfile(profAttrs);
 
             const userArg = mergedArgs.knownArgs.find((arg) => arg.argName === "user");
-            expect(userArg.argValue).toBeUndefined();
+            // expect(userArg.argValue).toBeUndefined(); // See zowe/imperative#755
+            expect(userArg.argValue).toBe("userNameBase"); // See zowe/imperative#755
```

Comment:
Since this PR fixes #755, should this change be reverted? The same comment applies for `passwordArg` below.

Reply:
I think the userNameBase (as well as the password) should be expected to be returned with `getSecureVals: false|undefined` since they are in plain text in the `__tests__/__resources__/ProfInfoApp_team_config_proj/ProfInfoApp.config.json`.

I could try to add one additional tests with a new project config that specifies them in the secure array.

Reply:
No need to add additional test, maybe its ok to just remove the comments from this file? I assumed they were TODO-related, but I also don't mind if you want to keep the reference to the issue 🙂

Reply:
I don't feel strongly about keeping the issue number there. 😋